endpoints/PushEndpoint.ts

```diff
@@ -0,0 +1,88 @@
+import { ApiEndpoint, IApiResponse, IApiRequest, IApiEndpointInfo } from "@rocket.chat/apps-engine/definition/api";
+import { IRead, IModify, IHttp, IPersistence } from "@rocket.chat/apps-engine/definition/accessors";
+import { isRegExp } from "util";
+import { RocketCaller } from "../utils/RocketCaller";
+
+const CACHE_KEY_ROOM_ID = "room_contact_uuid_"
+
+export class PushEndpoint extends ApiEndpoint {
+    public path = "push/webhook";   
+    
+    public async get(
+        request: IApiRequest,
+        endpoint: IApiEndpointInfo,
+        read: IRead, 
+        modify: IModify,
+        http: IHttp,
+        persis: IPersistence
+    ) : Promise<IApiResponse> {
+
+        const org_id = request.query.orgId;
+        const department_name = request.query.department;
+        let contact_uuid = request.query.contactUuid;
+        const priority = request.params.priority;
+        const visitor = this.getVisitorFromParams(request.query);
+        if (!contact_uuid) {
+            contact_uuid = request.query.token;
+        }
```

Comment:
We need to validate if that params exists, their types and formats, so we can respond the request with a proper message when it's  malformed, facilitating the user debugging.